### ISSUE TITLE
Remove deprecated properties from the API

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/network-pulse-api#241]
We have been adding new properties such as `creator_with_profiles` followed by `related_creator` instead of replacing existing ones in the serializers. Once the front-end and back-end are in sync, we should remove these properties to clean up the code.

**Comment by gideonthomas** [comment]
@alanmoo looks like this still needs to be removed:
https://github.com/mozilla/network-pulse-api/blob/e7127dcbc83896d5852055b03d5ef975b31605c4/pulseapi/entries/views.py#L479-L488

**Comment by alanmoo** [comment]
Good catch, I thought I searched through the repo when cleaning it up pre-Mozfest.

**Comment by ppatel221** [comment]
Hello, I'm a student currently enrolled in an open source class. Is it okay if I work on this?

**Comment by alanmoo** [comment]
Sure thing!

**Comment by ppatel221** [comment]
Ive submitted a PR. I've deleted the lines you specified. But the checks are failing on Travis. https://github.com/mozilla/network-pulse-api/pull/266

**Comment by alanmoo** [comment]
Looks like you’ve got some formatting problems, try running flake8 locally and it should tell you what’s wrong.

-Alan Mooiman,
Mozilla Foundation Engineering Manager
Eastern Standard Time (USA) UTC-5

**Comment by ppatel221** [comment]
I've ran Flake8 and i fixed the formatting errors and deleted some unused variables. It shows me these two errors still. 
**I went from:**
pulseapi/entries/views.py:16:1: F401 'pulseapi.creators.models.OrderedCreatorRecord' imported but unused
pulseapi/entries/views.py:16:1: F401 'pulseapi.creators.models.Creator' imported but unused
pulseapi/entries/views.py:258:17: E722 do not use bare except'
pulseapi/entries/views.py:434:13: E722 do not use bare except'
pulseapi/entries/views.py:439:13: F841 local variable 'creator_data' is assigned to but never used
pulseapi/entries/views.py:480:17: E303 too many blank lines (2)

**To:**
pulseapi/entries/views.py:257:17: E722 do not use bare except'
pulseapi/entries/views.py:433:13: E722 do not use bare except'

What should i do with these excepts

**Comment by ppatel221** [comment]
I just fixed all of the flake8 errors I was getting. After fixing it I submitted a PR and it still won't pass the test. https://github.com/mozilla/network-pulse-api/pull/267
